ros/src/common/libs/diagnostics_lib/diag_msgs/package.xml

```diff
@@ -0,0 +1,19 @@
+<package>
+  <name>diag_msgs</name>
+  <version>1.8.0</version>
+  <description>The libdiag package</description>
+
+  <maintainer email="masaya.kataoka@tier4.jp">Masaya Kataoka</maintainer>
+  <license>BSD</license>
+  <author email="yusuke.fujii@tier4.jp">Yusuke FUJII</author>
+  <author email="masaya.kataoka@tier4.jp">Masaya Kataoka</author>
+  <buildtool_depend>catkin</buildtool_depend>
```

Comment:
add build tool is ```autoware_build_flags```

Reply:
add <buildtool_depend>autoware_build_flags</buildtool_depend>
to my packages.